flocker/volume/test/test_service.py

```diff
@@ -613,12 +616,13 @@ def test_enumerate_skips_other_filesystems(self):
         service = VolumeService(FilePath(self.mktemp()), pool, reactor=Clock())
         service.startService()
 
-        name = VolumeName(namespace=u"mynspaces", id=u"good volume name")
+        name = VolumeName(namespace=u"mynspaces",
+                          dataset_id=u"good volume name")
```

Comment:
This value doesn't conform to the documented constraints (spaces are supposedly not allowed).